_activities/cwp.md

```diff
@@ -28,12 +28,26 @@ we are reaching out to other HEP experiments around the world to participate.
 
 ## Final Roadmap Paper
 
-The first draft of the [Roadmap for HEP Software and Computing R&D for the 2020s](https://docs.google.com/document/d/1rcPIJQc3LNAh5tjHKjfuq80StrMO5ksiLwhDlJzeg9U/edit?usp=sharing){:target="_cwp_roadmap"} 
-report has been released to the community for comment (archived [pdf](/assets/HSF-Community-White-Paper-v0.1.pdf), also available with [line numbers](/assets/HSF-Community-White-Paper-v0.1-linenumbers.pdf)).
+The aim is to produced the final CWP by December 15, 2017.
```

Comment:
produced -> produce (but actually just "The aim is finish the CWP" is more succinct.